Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerForMainServer.java

```diff
@@ -26,6 +26,11 @@ public boolean createSession(String name, String password, String hostName, int
         return GameServer.getInstance().getSessionController().createSession(name, password, hostName, capacity, duration, mapType, ballType);
     }
 
+    @Override
+    public int getAvailableMemory() throws RemoteException {
+        return (int) Runtime.getRuntime().freeMemory();
+    }
+
     public final void close() throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)